package-lock.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "connect-qos",
   "version": "5.1.1",
-  "lockfileVersion": 2,
+  "lockfileVersion": 3,
```

Comment:
I assume this was unintentional, double check you are using the proper node/npm version

Reply:
It's intentional. Consistent with `.nvmrc`